_dashboards/maps/maptiles.md

```diff
@@ -1,7 +1,8 @@
 ---
 layout: default
-title: WMS map server
-nav_order: 20
+title: Configure WMS map server
```

Comment:
Configuring WMS map server (map or maps?)

Reply:
Revised. Proper name is Web Map Service. Updated doc accordingly.